modules/javafx.swing/src/main/java/javafx/embed/swing/SwingNode.java

```diff
@@ -361,7 +365,9 @@ public JComponent getContent() {
      */
     private void setContentImpl(JComponent content) {
         if (lwFrame != null) {
-            swNodeIOP.disposeFrame(lwFrame);
+            rec.dispose();
```

Comment:
is it possible for `rec` to be null at this point?
would it make sense to rewrite this code block as
```
if(rec != null) {
  rec.dispose();
  rec = null;
}
if(disposeRecRef != null) {
  Disposer.removeRecord(...)
  disposerRecRef = null;
}
if(lwFrame != null) {
 lsFrame = null;
}
```
?

Reply:
If `lwFrame` is not null, little chance of `rec` to be null at this point as unlike PR#1178 there's no multiple threads involved, only EDT, so no race condition causing class variables to null out, 
but anyways I have added null check to be safe..